bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/i18n/tesla.properties

```diff
@@ -206,7 +206,7 @@ channel-type.tesla.minavailabletemp.label = Minimum Temperature
 channel-type.tesla.minavailabletemp.description = Indicates the minimal inside temperature of the vehicle
 channel-type.tesla.mobileenabled.label = Mobile Enabled
 channel-type.tesla.mobileenabled.description = Indicates whether the vehicle can be remotely controlled
-channel-type.tesla.notenoughpower.label = Not Enought Power
+channel-type.tesla.notenoughpower.label = Not Enough Power
```

Comment:
Please also update the typo in the XML file.

And as you added a new thing type, you need to add all the new information for this thing type in this file (the easiest way is to run the i18n tool).

Reply:
I always forget this. But I do not seem to be the only one - regenerating the files shows a hell of a lot new entries that are not all related to my changes. 😉